### ISSUE TITLE
Changing default Port 25 for SMTP 

### ISSUE THREAD

**Issue by ashiq54689** [mailcow/mailcow-dockerized#1675]
I am hosting Mailcow in GCP. And by default, they block Port 25. What option do I have in this case? Is it possible to change the default Port 25 to 587 or something else? Currently, I am getting Connection timed out error.

`postfix-mailcow_1    | Aug 16 02:07:35 mail2 postfix/smtp[568]: D16155F424: to=<username@gmail.com>, relay=none, delay=7518, delays=7426/0.02/92/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[173.194.219.27]:25: Connection timed out)`

**Comment by mkuron** [comment]
You need port 25 incoming to receive messages from other servers. Without it, you can only fetch messages from another server via imapsync jobs, which can be configured per user.

You need port 25 outgoing to send messages to other servers. Without it, you need to send your messages via a relayhost, which can be configured in the admin UI.

So a mail server without TCP port 25 is rather limited. Usually you can ask your hoster to unblock it for you though.

**Comment by ashiq54689** [comment]
Incoming is fine. So I guess, Port 25 is open for incoming, only outgoing is blocked. 

And, no, they won't unblock it, Port 25 is blocked in GCP permanently. So, it seems, Mailcow is broken in Google Cloud!

**Comment by ashiq54689** [comment]
In mailcow.conf, there is an option SUBMISSION_PORT=587. So how do I enable the submission Port? That will serve my purpose.

**Comment by andryyy** [comment]
It is enabled and always bound by default. :-)

If port 25 is closed, it is useless anyway.

**Comment by ashiq54689** [comment]
Alright, it seems Outlook is working using 587. This will work for now. I understand relay is one option. But is there any other way for SOGo to send Email using some other port? Or that's simply impossible?

**Comment by andryyy** [comment]
SOGo sends mail via postfix-mailcow on port 588.

**Comment by mkuron** [comment]
SOGo communicates internally and won't be affected by these rules. Only messages leaving your server will be blocked if you don't use a relayhost. This is not Mailcow-specific but would affect you with any mail server software you tried to run on GCP.

**Comment by ashiq54689** [comment]
Thanks for the clarification. I am back to SoftLayer/IBM Cloud. It has port 25 open.

**Comment by RogerSik** [comment]
Is it possible to have addtional a port? We need it for some server which should in future use mailcow as relay server.  The hoster of the other said server has 25 blocked.

**Comment by mkuron** [comment]
You can only have one SMTP port. However, you can create an account on your Mailcow and tell the other server to submit messages to port 587 with that account.

**Comment by clairelim97** [comment]
@mkuron How did you tell the other server to submit messages at port 587 with that account? Is it under routing > sender dependent transport? 

I created a sender-dependent transport and added the host as my IP:587 but it still doesn’t seem to work

**Comment by mkuron** [comment]
If you are referring to relaying outgoing emails via another server: you should indeed specify a port number for relay hosts and for sender-dependent transports. Have a look at the postfix logs to see what's going on exactly, but if you need help you are better off asking on the forum or telegram channel. It's probably not a bug.

**Comment by javier-marroquin** [comment]
The same thing is happening to me, Vultr has blocked port 25 permanently. Apparently Linode too, this will be a problem. I'll look for another provider, but there should be a solution to this.

**Comment by andryyy** [comment]
We will create a petition.
